Modules/Core/Common/include/itkCopy.h

```diff
@@ -0,0 +1,46 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#ifndef itkCopy_h
+#define itkCopy_h
+
+namespace itk
+{
+
+/** Returns a copy of its argument. Primarily used to make the act of copying (typically involving a copy-constructor
```

Comment:
Spelling error somewhere on this line?

Reply:
Maybe @dave3d can tell us? 🤷

Reply:
Might be:
Coverity, Intellisense, Synopsys, lnt

Reply:
Thanks @issakomi, Now I see, those four terms are mentioned in the CI 
output, at https://github.com/InsightSoftwareConsortium/ITK/actions/runs/7400527348/job/20134312236

```

    Modules/Core/Common/include/itkCopy.h, 25
    Modules/Core/Common/include/itkCopy.h, 25
    Modules/Core/Common/include/itkCopy.h, 25
    Modules/Core/Common/include/itkCopy.h, 25

Coverity:

Intellisense:

Synopsys:

lnt:

4 misspellings found
```

I hope Dave (@dave3d) can still make the CommentSpellCheck output more readable. I would think that `Modules/Core/Common/include/itkCopy.h, 25` is about line 25 in that source file, but apparently it's not, or is it? It's not clear to me either why those misspellings end with a colon (`:`). What that intentional?

Anyway, I'll add those to the dictionary file, https://github.com/InsightSoftwareConsortium/ITK/blob/d45127b5a3296b7f1721b57e0ffe8e0071d0c64f/.github/workflows/itk_dict.txt

Reply:
I'll check the code out today.  Also I plan to improve the spell checking output.  Probably next week.

Reply:
@dave3d Cool, thanks!